src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/PathGradientBrush.cs

```diff
@@ -188,15 +188,17 @@ public Blend Blend
                 // The Desktop implementation throws ArgumentNullException("source") because it never validates the value of value.Positions, and then passes it
                 // on to Marshal.Copy(value.Positions, 0, positions, count);. The first argument of Marshal.Copy is source, hence this exception.
                 if (value.Positions == null)
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                     throw new ArgumentNullException("source");
+#pragma warning restore CA2208
```

Comment:
Same question

Reply:
Because of the commen above.

I am updating the pragma comments with the reason of suppressing